processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -110,6 +110,21 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
   }
 
   @Override
+  public boolean hasMultipleValues(final String dimension)
+  {
+    if (isFilterableVirtualColumn(dimension)) {
```

Comment:
This check is here to match the checks in the other BitmapIndexSelector methods. All of them have a check that looks like:

```java
if (isFilterableVirtualColumn(dimension)) {
  // treat "dimension" as a virtual column
  return something;
}

// treat "dimension" as a regular column
```